README.md

```diff
@@ -40,11 +42,51 @@ The Battery Toolkit “Commands“ menu and its menu bar extra allow you to issu
 |:--:| 
 | *Battery Toolkit Menu Bar Extra* |
 
+# Install
+
+### Manual Install
+1. Go to the [releases](https://github.com/mhaeuser/Battery-Toolkit/releases/latest)
+2. Download the newest non-dSYM build (e.g. `Battery-Toolkit-1.6.zip`)
+3. Unzip it (double click)
+4. Drag it into your Applications folder
+5. Right click the `Battery Toolkit.app`, then click "Open", then allow in the dialog box
+6. If that doesn't work, go to `System Settings -> Privacy & Security` and you should see a prompt at the bottom of the screen allow opening Battery Toolkit. Allow it.
+7. Open Battery Toolkit again from Applications folder.
```

Comment:
Sorry for the nitpicks, but some bullets end with periods and some don't. :) Should probably adjust existing bullets as well.